sqlgen/db.go

```diff
@@ -92,6 +92,23 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 	return db
 }
 
+func (db *DB) BaseCount(ctx context.Context, query *BaseCountQuery) (int64, error) {
```

Comment:
Is there any reason we want this to be public API?

Reply:
+godoc

Reply:
unexported.